src/main/java/frc/robot/commands/Climb.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.commands;
+
+import com.spikes2212.dashboard.RootNamespace;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+import java.util.function.Supplier;
+
+public class Climb extends CommandBase {
+
+    private static final RootNamespace namespace = new RootNamespace("climb");
+    private static final Supplier<Double> preClimbSpeed = namespace.addConstantDouble("pre climb speed", 0.6);
+    private static final Supplier<Double> midClimbSpeed = namespace.addConstantDouble("mid speed climb", 0.3);
+    private static final Supplier<Double> preClimbTolerance = namespace.addConstantDouble("pre climb tolerance", 15);
+    private static final Supplier<Double> midClimbTolerance = namespace.addConstantDouble("mid climb tolerance", 3);
+    private static final Supplier<Double> yawSetpoint = namespace.addConstantDouble("yaw setpoint", 75);
```

Comment:
change to 90 and create a sequencial command group, with the climbing as the first command and a pid based rotation as the second